xbmc/platform/darwin/ios/XBMCController.mm

```diff
@@ -41,23 +49,15 @@
 
 #include <sys/resource.h>
 
+#import <AVFoundation/AVAudioSession.h>
```

Comment:
wrong include order, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
Is this not "Other Libraries"?

Reply:
It is, but `sys/resource.h` is also other libraries.

Reply:
no probs